VisualStudio/Installer/source.extension.vsixmanifest

```diff
@@ -1,9 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
     <Metadata>
-        <Identity Id="874C448B-42DE-466B-A7E7-0EFEC1E7C009" Version="0.0.9.2" Language="en-US" Publisher="Open Ria Services" />
+        <Identity Id="874C448B-42DE-466B-A7E7-0EFEC1E7C009" Version="0.0.9.3" Language="en-US" Publisher="Open Ria Services" />
         <DisplayName>Open Ria Services</DisplayName>
-        <Description xml:space="preserve">sOpenRiaServices Tooling and Templates For Visual Studio 2015, 2017 and 2019
+        <Description xml:space="preserve">sOpenRiaServices Tooling and Templates For Visual Studio 2017+ and 2019
```

Comment:
What does the + do? or is it like versions above 2017 but then you wouldnt need 2019?

Reply:
Yes either 2019 or the "+" should be removed